### ISSUE TITLE
Locale support

### ISSUE THREAD

**Issue by c-sanchez** [nightmare-space/speed_share#5]
Hi, SpeedShare looks really cool and useful, however it is a pity that the program is not available in more languages, both the mobile and desktop applications.

Add support for translations, I would like to help with Spanish translation if possible.

By the way, I think it would be better if the name of the application is displayed in English for everyone, SpeedShare.
It's a bit strange to see characters that I don't understand heh.

**Comment by mengyanshou** [comment]
first of all thank you very much,but I've been a little busy lately. Internationalization will be introduced in about a month,thanks again~

**Comment by c-sanchez** [comment]
Hi @mengyanshou, no worries, thanks for your reply, great to know that we will have this app available in more languages :D
Will you update the desktop version as well?

**Comment by mengyanshou** [comment]
Yes, since this project was developed using Flutter, any changes will apply to all platforms,And I've changed my mind. I'm going to join internationalization sometime this week~

**Comment by c-sanchez** [comment]
Hola mengyanshou, how are you?
I am following the project, I saw that a few days ago you uploaded some code related to translation support.

**Comment by mengyanshou** [comment]
Ok~, you can create a Spanish translation file based on the 'intl_en.arb' file, when finished, submit the PR to this project.

**Comment by mengyanshou** [comment]
It's great! Now I have initially added Spanish into the project, but the coverage of internationalization is not complete, and there is no caching, which needs to wait for the later development. I submitted the code to the dev branch, here are some download links
[arm64](http://nightmare.fun/YanTool/resources/SpeedShare/%E9%80%9F%E4%BA%AB_2.0.3-dev_Android_arm64.apk)
[armv7](http://nightmare.fun/YanTool/resources/SpeedShare/%E9%80%9F%E4%BA%AB_2.0.3-dev_Android_arm_v7a.apk)

**Comment by c-sanchez** [comment]
Yes, there are still some strings to translate, but it's a good start, thanks for the binaries.
Please leave me a message here when there is an update :)

**Comment by c-sanchez** [comment]
Hi @mengyanshou , Spanish locale update :D

```
{
    "common": "Común",
    "setting": "Ajustes",
    "downlaodPath": "Ruta de descarga",
    "autoDownload": "Descarga automática",
    "clipboardshare": "Compartir portapapeles",
    "messageNote": "Vibrar al recibir un mensaje",
    "aboutSpeedShare": "Acerca de Speed Share",
    "currenVersion": "Versión actual",
    "otherVersion": "Descargar otra versión",
    "downloadTip": "SpeedShare también es compatible con Windows, macOS y Linux",
    "lang": "Idioma",
    "chatWindow": "Ventana de chat",
    "developer": "Desarrollador",
    "openSource": "Código abierto",
    "noMessageTip": "Actualmente no hay mensajes, haga clic para entrar en la lista de mensajes",
    "inputConnect": "Ingresar conexión",
    "scan": "Escanear código QR",
    "log": "Registro",
    "theTermsOfService": "Términos de servicio",
    "privacyAgreement": "Acuerdo de privacidad",
    "qrTips": "Deslice hacia un lado para cambiar de IP",
    "ui": "Diseñador de interfaz",
    "recentFile": "Archivos recientes",
    "recentImg": "Imágenes recientes",
    "currentRoom": "Sala de chat",
    "remoteAccessFile": "Archivo local de acceso remoto",
    "remoteAccessDes": "Puede administrar los archivos locales abriendo esta IP en el navegador.",
    "directory": "Directorio",
    "unknownFile": "Archivo desconocido",
    "zip": "Zip",
    "doc": "Documento",
    "music": "Música",
    "video": "Vídeo",
    "image": "Imagen",
    "apk": "Apk"
}
```

Missing strings to transle:

main window
https://i.ibb.co/BPMfm3D/speedshare-main-window.jpg

Files type window
https://i.ibb.co/tZb3rYj/speedshare-files-type-window.jpg

input ip window
https://i.ibb.co/x6VxyYN/speedshare-ip-window.jpg

text input field from chat window
https://i.ibb.co/YbwHLcG/speedshare-text-input-chat-window.jpg

text input field context menu from chat window
https://i.ibb.co/XWMZjQ1/speedshare-text-input-context-menu-chat-window.jpg

Program name
https://i.ibb.co/KbJV9yr/speedshare-android-launcher-icon.jpg
Yeah, at least for me is a bit weird to have Speed Share icon in my Android launcher with Chinese text hehe

**Comment by mengyanshou** [comment]
okk!
